package.json

```diff
@@ -80,7 +80,7 @@
     "react-native-simple-markdown": "^1.1.0",
     "react-native-svg": "12.1.1",
     "react-native-swipe-gestures": "^1.0.5",
-    "react-native-tuvali": "github:mosip/tuvali#v0.4.3",
+    "react-native-tuvali": "github:mosip/tuvali#v0.4.4",
```

Comment:
please do run npm i && ios && pod install also along with this

Reply:
Have updated the package-lock.json